packages/node-core/src/indexer/storeCache/storeModel.service.ts

```diff
@@ -19,19 +20,29 @@ export interface IStoreModelService {
   poi: IPoi | null;
   metadata: IMetadata;
 
+  init(historical: boolean, useCockroachDb: boolean, meta: MetadataRepo, poi?: PoiRepo): void;
```

Comment:
This shouldn't be on the interface in the same way the constructor isn't on the interface.